test/index.test.js

```diff
@@ -500,16 +501,13 @@ describe('mongoose module:', function() {
   });
 
   it('throws an error on setting invalid options (gh-6899)', function() {
-    let threw = false;
     try {
       mongoose.set('someInvalidOption', true);
+      assert.fail('Expected mongoose.set to throw');
```

Comment:
Can you replace this with `assert.throws()` instead please?

Reply:
sure, should other tests also be re-written to use this? (i based my code style on the other tests (just removed the unnecessary `let threw = false`)

Reply:
i just tried to do this, but it seems like `assert.throws` only supports checking that it is a instance of a error and the message, but does not return the error anywhere to check additional properties, which is used in the current version of the new tests to check each error (and the `errors` object) individually

Reply:
That's a good point, we can't use `assert.throws()`. Please disregard this comment.